src/static/js/wishlist.js

```diff
@@ -91,3 +91,45 @@ setTimeout(() => {
   document.querySelectorAll('.upForm').forEach(element => { listen(element, 'up') })
   document.querySelectorAll('.downForm').forEach(element => { listen(element, 'down') })
 }, 0)
+
+function priceToFloat(price) {
+  let cleanPrice = price
+      .replace(/[^\d.,]/g, '')  // Remove everything except digits, periods, and commas
+      .replace(/,/g, '');       // Remove commas to standardize the number
+  // If there's more than one decimal separator (.), treat the last one as the decimal point
+  if ((cleanPrice.match(/\./g) || []).length > 1) {
+      cleanPrice = cleanPrice.replace(/\.(?=.*\.)/, '');  // Remove the first period if there are multiple
+  }
+  return parseFloat(cleanPrice);
+}
+
+document.addEventListener('DOMContentLoaded', function(){
+  let sortOrderPrice = 'asc';
+  const priceArrow = document.getElementById('price-arrow');
+  const sortByPrice = document.getElementById('sort-price');
+
+  function updateArrow(column, order) {
+    if (column === 'price') {
+      priceArrow.className = order === 'asc' ? 'fas fa-arrow-up' : 'fas fa-arrow-down';
+    }
+  }
+
+  sortByPrice.addEventListener('click', function(e){
+    e.preventDefault();
+
+    const table = document.getElementById('wishlist-table');
+    const tbody = table.querySelector('tbody');
+    const rows = Array.from(tbody.querySelectorAll('tr'));
+    sortOrderPrice = sortOrderPrice === 'asc' ? 'desc' : 'asc';
+
+    const sortedRows = rows.sort((a,b) => {
```

Comment:
This is prone to breaking if I add another column. Can you use `<a/b>.querySelector('.price').textContent` and add a .price here?

https://github.com/cj13579/Christmas-Community/blob/9e366c870fc0248cdb215611f0fc808954276058/src/views/wishlist.pug#L89

Reply:
Yep, that's a good shout. Fixed with this commit: https://github.com/Wingysam/Christmas-Community/pull/140/commits/cab10ce3744eb02333649aca5dfce96918da773b